projects/swimlane/ngx-ui/src/lib/components/select/select-dropdown.component.html

```diff
@@ -1,79 +1,96 @@
 <div>
-  <div class="ngx-select-filter" *ngIf="filterable && !tagging">
-    <i class="ngx-icon ngx-search"></i>
+  <div class='ngx-select-filter' *ngIf='filterable && !tagging'>
```

Comment:
It seems the prettier didn't run as we switched to from double quotes to single quotes on several of the files...Can we take a look at this?